LinkedList.java

```diff
@@ -0,0 +1,206 @@
+package linkedListFunctions;
+
+public class LinkedList {
+	Node head;
+
+	class Node {
+		int data;
+		Node next;
+
+		Node(int d) {
+			data = d;
+			next = null;
+		}
+	}
+
+	public void insertMiddle(int data, int toInsert) {

```

Comment:
Please use proper variable names : Use target instead of data; 2. data or x instead of toInsert

Reply:
okay